packages/payment-processor/package.json

```diff
@@ -39,6 +39,7 @@
     "test:watch": "yarn test --watch"
   },
   "dependencies": {
+    "@openzeppelin/contracts": "4.7.3",
```

Comment:
The superfluid framework does not implement the erc777 interface. I had to get the abi from here.